rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -145,7 +145,7 @@
         'https://poap.delivery/proof-of-gucci-design',
         'YFI - Proof of Gucci Design Competition',
     ),
-    'resucitators': (
+    'resuscitators': (
```

Comment:
Since you are touching the python file. There seems to be an error in line 82 and 92. The names seem to be wrong and need to be swapped?